src/manage/dashboard/index.md

```diff
@@ -1,71 +1,110 @@
+
+You can learn more about project history and how to revert to an older version in [Project History and Versions](../project-details/).
+
+### Collaborators
+The **Collaborators** tab provides the list of users who have access to the project as well as pending invitations. You can use the **Share** button to invite more people to contribute to your project. 
+
+Here, the [**project permissions**](../permissions/#project-permissions) of current collaborators can be changed.
+
+![Mergin Maps project collaborators](./mergin-maps-dashboard-project-collaborators.jpg "Mergin Maps project collaborators")
+
+### Settings
+In **Settings**, you can:
+- make the project [public or private](../project-advanced/#make-your-project-public-private)
+- [Transfer project](../project-advanced/#transfer-a-project) to another workspace
+- [Delete project](../project-advanced/#delete-a-project) 
+
+![Project settings in Mergin Maps Dashboard](./mergin-maps-dashboard-project-settings.jpg "Project settings in Mergin Maps Dashboard")
 
-![Mergin Maps Web Map feature](./mergin-map.jpg "Mergin Maps Web Map feature")
 
 ## Members
-The **Members** tab provides the overview of the [members and guests](../permissions/#workspace-members-and-guests) of the workspace. Here, you can [add users](../project-advanced/#add-users-to-a-workspace) to your workspace and manage their [permissions](../permissions/).
-![Mergin Maps web workspace members](./web-members.jpg "Mergin Maps web workspace members")
+The **Members** tab provides the overview of the [members and guests](../permissions/#workspace-members-and-guests) of the workspace, as well as pending invitations. 
+Here, you can [**invite**](../project-advanced/#add-users-to-a-workspace) users to the workspace and manage the [roles](../permissions/#workspace-member-roles) of your workspace members. If needed, users can be also removed from the workspace.
+
+![Mergin Maps dashboard workspace members](./mergin-maps-dashboard-members.jpg "Mergin Maps dashboard workspace members")
+
+## Subscriptions
+The **Subscriptions** tab contains information about your [subscription plan](../subscriptions/).
 
-## Subscription
-The **Subscription** tab contains information about your [subscription plan](../subscriptions/).
+Here, you can find the current usage of storage, number of users (members and guests) and number of projects in your workspace.
 
-Here, you can find the current usage of storage, number of projects and number of users (members and guests) in your workspace.
+![Mergin Maps Subscriptions details](./mergin-maps-dashboard-subscriptions.jpg "Mergin Maps Subscriptions details")
 
-![Mergin Maps Subscriptions details](../subscriptions/subscriptions.jpg "Mergin Maps Subscriptions details")
+Click on **Manage subscription** to visit the subscription management portal where you can update the subscription plan, billing information, see invoices and upcoming payments. 
```

Comment:
Edit text:
Click on **Manage subscription** to visit the subscription management portal where you can update the subscription plan, billing information, and see invoices and upcoming payments.